.github/workflows/pr-realm-js.yml

```diff
@@ -84,7 +84,7 @@ jobs:
       - name: Setup node version
         uses: actions/setup-node@v4
         with:
-          node-version: 20
+          node-version: 20.11.1
```

Comment:
This change can be reverted once https://github.com/prebuild/prebuild/pull/325 gets merged, release and we adopt the change.